week-2/14-order/readme.md

```diff
@@ -6,8 +6,11 @@ In this exercise, you'll use order to move the items in the list. Follow the ste
 
 1. Look at the First list of countries. Then look at the solution displayed to the side. Find the differences between the two lists.
 2. Use `order` to make the list of countries on the left look like the ones on the right.
+
 ---
+
 > **Remember:** if you forgot how to use the `order` property, you can always:
+>
 > 1. Google the word "flexbox"
 > 2. Click on the link from "CSS-tricks"
-> ![Screenshot of how to google flexbox](/images/14/googling-flexbox.png)
+>    ![Screenshot of how to google flexbox](/(/images/14/googling-flexbox.png)
```

Comment:
Again, check what you're adding to your commits using `git status` and `git diff` :slightly_smiling_face: . If you find you've changed something by mistake (like  this readme file), then you can undo the changes, either manually or using git (`git checkout -- week-2/14-order/readme.md` for this file). Then `git status` would no longer show this file as being changed.